### PR TITLE
Test on Travis.

### PR DESCRIPTION
**Pull request by dpoirier**

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -1,3 +1,10 @@
+#
+# IMPORTANT NOTE: This .travis.yml file is intended for
+# testing the project template itself. If you want to
+# test a project created from the project template,
+# please rename "project.travis.yml" to ".travis.yml",
+# overwriting this file.
```
Comment (ironfroggy) [inline .travis.yml]: Can we note that the rename is already done if they run `make setup`?

Comment (dpoirier) [inline .travis.yml]: Sure, will add that to the comment.

File: README.rst [hunk 2]
```diff
@@ -83,6 +83,12 @@ Create the Postgres database and run the initial migrate::
     ({{ project_name }})$ createdb -E UTF-8 {{ project_name }}
     ({{ project_name }})$ python manage.py migrate
 
+If you want to use `Travis <http://travis-ci.org>`_ to test your project,
+rename ``project.travis.yml`` to ``.travis.yml``, overwriting the ``.travis.yml``
```
Comment (ironfroggy) [inline README.rst]: also note `make setup` here

Comment (dpoirier) [inline README.rst]: this is in the "manual steps that 'make setup'" would do for you" section of the README, right?

**Comment by ironfroggy** [comment]
seeing the passing travis test, with the small comment/doc updates I say +1 and awesome work!

File: README.rst (hunk 2 above)
Comment (ironfroggy) [inline README.rst]: oh right right nevermind

**Comment by vkurup** [comment]
Looks good to me too! :+1: assuming the travis failure is a fluke

**Comment by ironfroggy** [comment]
:rabbit: and okay with clean / rebase as long as we wait for the builds to clear again

File: .travis.yml [hunk 3]
```diff
@@ -9,32 +17,34 @@ python:
 addons:
   postgresql: "9.3"
 
+# In general, cache things where they are. The only cached directory
+# we can't do that for is node_modules because it doesn't exist until
+# after we run 'start_project', so after that we symlink to the cached
```
Comment (vkurup) [inline .travis.yml]: s/start_project/startproject/

**Comment by vkurup** [comment]
:+1:

**Pull request merged**